src/components/Theme/style.scss

```diff
@@ -1,5 +1,14 @@
+@use '~@onfido/castor';
 @import './constants';
 
+:global {
+  @include castor.Button();
+}
+
+:root {
+  @include castor.day();
+}
+
```

Comment:
I wonder if using the global scope might be an issue for integrators. I can see that the variables have very specific names so I don't think there can be issue there but I'm leaving this here as an open question.

Reply:
This did end up causing a bug. 😅

- https://github.com/onfido/onfido-sdk-ui/issues/1804